### DISCUSSION TITLE
Subscript caracters implementation

### DISCUSSION THREAD

**Discussion by NicoFerra** [gsantner/markor#2018] [Ideas] [answered]
Hey everyone, I've been using Markor as my main note taking app for half a year now and I love it.  
The only feature I miss from other apps (e.g. I use [Apostrophe](https://github.com/ApostropheEditor/Apostrophe) on desktop), is the extended syntax for subscript characters: I often write molecule formulas for work and I'd love to have a quick way of doing it and to have compatibility with files I wrote on PC.  
Since in Markor the syntax for superscript characters already follows the guidelines from the Markdown Guide, [subscript](https://markdownguide.org/extended-syntax/#subscript) characters should follow them too and use a **tilde** symbol (\~), so
```
H~2~O
``` 
would become **H<sub>2</sub>O**, instead of having to input the whole 
```html
H<sub>2</sub>O
```
every time.

In any case thanks for all the work you are putting in this app, it's hands down the best on Android.

**Comment by gsantner** [comment]
Markor uses a Markdown parser but not implements one of it own. Maybe you can find something useful here at Markor Markdown parser - https://github.com/vsch/flexmark-java/wiki/Extensions.

Alternative, for math & formulas KaTex might be also something for you, can be enabled in settings.

**Reply by NicoFerra** [reply, marked as answer]
Apparently subscript formatting is already within the parser extensions and can be enabled [this way](https://github.com/vsch/flexmark-java/wiki/Extensions#gfm-strikethroughsubscript).

**Reply by gsantner** [reply]
@NicoFerra  
Nice, it also seems quite easy to add this. Would be happy about getting contributions. So if you want please feel free to add it and create a merge request. Always happy to welcome new contributors, there is a very low amount of people contributing but high amount of people asking for things 😄 .

**Reply by NicoFerra** [reply]
I'm gonna try, I still have to learn GitHub (and programming in general 😅 ) but I'd love to partecipate!
